src/QsCompiler/LanguageServer/Program.cs

```diff
@@ -102,7 +103,16 @@ private static int Run(Options options)
             // This needs to be done before any MsBuild packages are loaded.
             try
             {
-                MSBuildLocator.RegisterDefaults();
+                VisualStudioInstance vsi = MSBuildLocator.RegisterDefaults();
```

Comment:
I would add a comment explaining why you are doing this and pointing to the github issue where you found the workaround.

Reply:
Sounds good. Adding.